### ISSUE TITLE
Make plot_code a Parameter

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/hvplot#885]
## Request

Change `plot_code` from a function to a parameter. Or document how to automatically update the plots code in some output cell in a notebook.

## Motivation

I love that the fact that I can export my `explorer` work as code.

But I would like the code to be able to just update automatically because

- Its friction that I have to rerun a cell to get the code.
- The output of the `plot_code()` might not be in sync with the explorer selections if I ran that function some time ago.
- I want to be able to easily deploy an explorer Panel data app to my users such that the starting point for a visualization is just a web page that can take us a long way, then we export the code and continue if/ when needed.

![image](https://user-images.githubusercontent.com/42288570/187124363-ec427ffe-1bbb-4d27-869f-e854ec79af33.png)

**Comment by jbednar** [comment]
Another approach I've suggested is to have a button to copy the code for the current plot to the clipboard; would that be useful in your case?

**Comment by MarcSkovMadsen** [comment]
That would also be great and solve my use cases, if it was part of the explorer.

**Comment by maximlt** [comment]
Yes adding a button is a good idea (a button to save the plot as a file too, I think that was the original suggestion).

**Comment by philippjfr** [comment]
Right, an export tab would be nice with options to export to file and to clipboard.
